docs/mkdocs/docs/api/macros/json_use_implicit_conversions.md

```diff
@@ -27,6 +27,12 @@ By default, implicit conversions are enabled.
     You can prepare existing code by already defining `JSON_USE_IMPLICIT_CONVERSIONS` to `0` and replace any implicit
     conversions with calls to [`get`](../basic_json/get.md).
 
+!!! hint "CMake option"
+
+    Implicit conversions can also be controlled with the CMake option
+    [`JSON_ImplicitConversions`](../../integration/cmake.md#json_legacydiscardedvaluecomparison)
+    (`ON` by default) which defines `JSON_USE_IMPLICIT_CONVERSIONS` accordingly.
```

Comment:
This is now duplicated on this page. I haven't decided yet whether to keep or remove the original at the top (see line 10).

Reply:
I would remove the hint above.